admin/src/components/dashboard.js

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from 'react';
```

Comment:
I would like to see some project structure here. In this project all components are under the `components` folder, which puts all of them in the same level of importance. Dashboard in this case is one of the main entry points, and it complies with the role of a container. Having a folder containing all this "entry points" will be an improvement for the a developer joining the project.